tests/test_utils.py

```diff
@@ -1,35 +1,89 @@
-# import pytest
-# from agoradatatools.etl import utils
-# from synapseclient import Synapse
+import pytest
+from agoradatatools.etl import utils
+from unittest.mock import patch
+import synapseclient
+import yaml
+import io
+import builtins
 
```

Comment:
Nit: Lets try to follow pep8 standards: imports should follow a certain order and be alphabetized: https://peps.python.org/pep-0008/#imports.  

> Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 1. Related third party imports.
> 1. Local application/library specific imports.
>
> You should put a blank line between each group of imports.

Reply:
Is there a setting in VSCode that will autoformat imports? I have black running to autoformat my code, but it does not seem to work on imports.

Reply:
There is a module called `isort`: https://marketplace.visualstudio.com/items?itemName=ms-python.isort that I think you can use.